README.md

```diff
@@ -1,101 +1,20 @@
-[![Build Status](https://travis-ci.org/MicheleBertoli/react-gmaps.svg?branch=master)](https://travis-ci.org/MicheleBertoli/react-gmaps)
+# React GMaps
 
-React Gmaps
-===========
+A modern Google Maps integration for ReactJS
 
-A [Google Maps](https://developers.google.com/maps/documentation/javascript/) component for [React.js](http://facebook.github.io/react/)
+## Features
 
-Features
---------
+- Marker, Polyline, Polygon components and more
+- Hook based
```

Comment:
Hooks?

Reply:
i think grammatically, it should be singular
at least by what i can find on linguee
singular:
https://www.linguee.com/francais-anglais/search?source=auto&query=plant+based
plural:
https://www.linguee.com/francais-anglais/search?source=auto&query=plants+based